telemetry/roles/prometheus_gaudi/tasks/main.yml

```diff
@@ -13,8 +13,11 @@
 # limitations under the License.
 ---
 
-- name: Check if Gaudi Prometheus metric exporter need to be deployed
-  when: hostvars['127.0.0.1']['prometheus_gaudi_support']
+- name: Check if telemetry entry is present in software_config.json
+  when: hostvars['127.0.0.1']['telemetry_entry_present']
```

Comment:
we have fix in internal repo 
![image](https://github.com/user-attachments/assets/d8d69d3b-a308-4cf5-8068-ef9bed52a9d1)